packages/core/package.json

```diff
@@ -63,6 +76,10 @@
     "lint": "eslint .",
     "test": "jest",
     "build": "bunchee",
+    "postbuild": "pnpm run copy && pnpm run modify-package-json",
+    "copy": "cp -r package.json CHANGELOG.md ../../README.md ../../LICENSE examples src dist/",
+    "modify-package-json": "node ./scripts/modify-package-json.mjs",
+    "prepublish": "echo \"please cd ./dist and run pnpm publish\" && exit 1",
```

Comment:
@himself65 does this work with changeset publish?

Reply:
yes